toonz/sources/common/ttest/ttest.cpp

```diff
@@ -168,7 +168,7 @@ void TTest::runTests(string name)
 	cout << "Test file : '" << testFile << "'" << endl;
 	char buffer[1024];
 	while (is.getline(buffer, sizeof(buffer))) {
```

Comment:
Although not directly related to this PR, using `std::getline` (and read into a `std::string`) is probably better here.